### ISSUE TITLE
Issues with storyshots

### ISSUE THREAD

**Issue by michaelgeorgeattard** [storybooks/storybook#3288]
A couple of issues I found with storyshots addon (latest unstable release):

1. 

Initializing as follows to use imageSnapshot does not work:

```
import initStoryshots, { imageSnapshot } from "@storybook/addon-storyshots";

initStoryshots({
    suite: "Image storyshots",
    test: imageSnapshot
});
```

I fixed this problem as follows:

```
initStoryshots({
    suite: "Image storyshots",
    test: imageSnapshot({})
});
```

If you confirm this issue, I can create a PR to update the docs or attempt to fix the limitation.

2.

Another problem I found is `imageSnapshot` timing out when trying to generate a snapshot on the playground running in the background. I found the culprit in `@storybook/addon-storyshots/src/test-body-image-snapshot.js`:

```
return page
  .goto(url, { waitUntil: 'networkidle0' })
```

If I remove the `{ waitUntil: 'networkidle0' }` option things work fine. Would it be possible to use some other `waitUntil` parameter, or atleast allow the consumer to pass in options?

**Comment by thomasbertet** [comment]
@michaelgeorgeattard thanks for taking the time to trying out the new release & report issues :) 
I believe we could remove the `networkIdle0` as default, and let the user pass in options yes. I'll do it if you don't mind.

Also, if you want contribute to the docs to improve it, please do ! I believe there is room for improvement there :+1:

**Comment by danielduan** [comment]
@thomasbertet is `test: imageSnapshot({})` with the `({})` correct or incorrect?

I think that's the only thing left in this issue to address. We can close after.

**Comment by Hypnosphi** [comment]
`networkidle0` fix released as [`3.4.0-rc.4`](https://github.com/storybooks/storybook/releases/tag/v3.4.0-rc.4)
